xml/ha_acl.xml

```diff
@@ -218,427 +172,569 @@
-   <para>
-    For example, consider the following XPath:
-   </para>
-<screen>//*[@id="rsc1"]</screen>
-   <para>
-    which locates all the XML nodes with the ID <literal>rsc1</literal>.
-   </para>
-   <para>
-    The abbreviated syntax is written like this:
-   </para>
-<screen>ref:"rsc1"</screen>
-   <para>
-    This also works for constraints. Here is the verbose XPath:
-   </para>
-<screen>//constraints/rsc_location</screen>
-   <para>
-    The abbreviated syntax is written like this:
-   </para>
-<screen>type:"rsc_location"</screen>
-   <para>
-    The abbreviated syntax can be used in &crmsh; and &hawk2;. The CIB
-    daemon knows how to apply the ACL rules to the matching objects.
-   </para>
-  </sect2>
- </sect1>
- 
-  <sect1 xml:id="sec-ha-acl-config-hawk2">
-  <title>Configuring ACLs with &hawk2;</title>
   <para>
-   The following procedures show how to configure read-only access to the
-   cluster configuration by defining a <literal>monitor</literal> role and
-   assigning it to a user. Alternatively, you can use &crmsh; to do so,
-   as described in <xref linkend="pro-ha-acl-crm"/>.
+   Before you can start configuring ACLs, you need to
+   <emphasis>enable</emphasis> use of ACLs. To do so, use the following
+   command in the &crmsh;:
   </para>
 
-  <procedure xml:id="pro-ha-acl-hawk2-role">
-   <title>Adding a Monitor Role with &hawk2;</title>
+  <screen>&prompt.root;<command>crm</command> configure property enable-acl=true</screen>
+  <para>
+   Alternatively, use &hawk2; as described in
+   <xref linkend="pro-ha-acl-enable-hawk2"/>.
+  </para>
+
+  <procedure xml:id="pro-ha-acl-enable-hawk2">
+   <title>Enabling Use of ACLs with &hawk2;</title>
    <step>
     <para>
-     Log in to &hawk2;: </para>
+     Log in to &hawk2;:
+    </para>
     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
    </step>
    <step>
     <para>
-     In the left navigation bar, select <guimenu>Roles</guimenu>. 
+     In the left navigation bar, select <guimenu>Cluster Configuration</guimenu>
+     to display the global cluster options and their current values.
     </para>
    </step>
    <step>
     <para>
-      Click <guimenu>Create</guimenu>.
+     Below <guimenu>Cluster Configuration</guimenu> click the empty drop-down box
+     and select <guimenu>enable-acl</guimenu> to add the parameter. It is added
+     with its default value <literal>No</literal>.
     </para>
    </step>
    <step>
     <para>
-     Enter a unique <guimenu>Role ID</guimenu>, for example,
-     <literal>monitor</literal>.
+     Set its value to <literal>Yes</literal> and apply your changes.
     </para>
    </step>
-   <step>
-    <para>
+  </procedure>
+
+ </sect1>
+
+ <sect1 xml:id="sec-ha-acl-create-ro-monitor-role">
+  <title>Creating a Read-Only Monitor Role</title>
+  <para>
+   The following subsections describe how to configure read-only access
+   by defining a <systemitem>monitor</systemitem> role both in &hawk2;
+   and &crmshell;.
```

Comment:
How about "either with hawk or crm shell"? So that users know they only need to configure it once with either of the tools.

Reply:
That's a good idea! :+1: Like that. Integrated.